lib/onboarding/screens/log-in.dart

```diff
@@ -0,0 +1,161 @@
+import 'dart:convert';
+
+import 'package:cboard_mobile/models/error.dart';
+import 'package:cboard_mobile/screens/home_screen.dart';
+import 'package:cboard_mobile/stylesheets/constants.dart';
+import 'package:cboard_mobile/onboarding/widgets/header.dart';
+import 'package:cboard_mobile/onboarding/widgets/username-field.dart';
+import 'package:cboard_mobile/onboarding/widgets/password-field.dart';
+import 'package:cboard_mobile/shared/app-bar.dart';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+
+// Not really sure how entrypted things should be here!
```

Comment:
@tim-nguyen-cs I tried looking at the web app to see how the encryption works. It appears that when I sign up, the password doesn't seem to be encrypted on the frontend? 
![image](https://user-images.githubusercontent.com/44158788/108132289-956cea80-7080-11eb-82ec-879d40ebf239.png)

After looking at this stackoverflow response, it says that password hashing should be done on the backend, so I don't think you need to do anything here. Please let me know your thoughts! https://security.stackexchange.com/questions/110948/password-hashing-on-frontend-or-backend

Reply:
I think this is an interesting point of conversation, and something we can definitely refine as we're building out our login architecture. I've referenced an article [here](https://stackoverflow.com/questions/3391242/should-i-hash-the-password-before-sending-it-to-the-server-side) and [here](http://ticki.github.io/blog/you-are-probably-doing-login-systems-wrong/) that talk about how we should implement client-side hashing, but so we can work that into our system as we develop our front-end or we can also see if we'd be able to integrate some kind of password encryption library that might help! 

Definitely sounds good though, I'll keep this open just so we have it on our radar - we should also probably make a ticket out of this so we don't forget.